react/Tooltip/Readme.md

```diff
@@ -1,8 +1,26 @@
 ### Simplest usage
 
-```js
+```jsx
+import Typography from 'cozy-ui/transpiled/react/Typography';
+import Icon from 'cozy-ui/transpiled/react/Icon';
+import CarbonCopyIcon from 'cozy-ui/transpiled/react/Icons/CarbonCopy';
 import Tooltip from 'cozy-ui/transpiled/react/Tooltip';
-<Tooltip title={'This is an explaination'}><u>hover over me</u></Tooltip>
+
+<Tooltip
+  {...(isTesting() && { open: true } )}
+  title={
+    <>
+      <div className="u-flex u-flex-items-center">
+        <Icon icon={CarbonCopyIcon} className="u-mr-half" />
+        <Typography variant="body1" color="inherit">Carbon copy</Typography>
```

Comment:
Adding a gutterBottom in the Typography would be nice I think.

Reply:
J'ai rajouté `u-mb-half` sur le parent sinon ça décentre le texte par rapport à l'icone